tests/llm/models/test_converse.py

```diff
@@ -0,0 +1,95 @@
+import base64
+import json
+from pathlib import Path
+
+from e84_geoai_common.llm.core.llm import (
+    Base64ImageContent,
+    LLMInferenceConfig,
+    LLMMessage,
+    TextContent,
+)
+from e84_geoai_common.llm.models.converse import BedrockConverseLLM
+from e84_geoai_common.llm.tests.mock_bedrock import (
+    converse_response_with_content,
+    make_test_bedrock_client,
+)
+
+
+def test_basic_usage() -> None:
+    llm = BedrockConverseLLM(
+        client=make_test_bedrock_client([converse_response_with_content("olleh")])
+    )
+    config = LLMInferenceConfig()
+    resp = llm.prompt(
+        [LLMMessage(content="Output the word hello backwards and only that.")], config
+    )
+    assert resp == LLMMessage(role="assistant", content=[TextContent(text="olleh")])
+
+
+def test_with_response_prefix() -> None:
+    llm = BedrockConverseLLM(
+        client=make_test_bedrock_client([converse_response_with_content("  15")])
+    )
+    config = LLMInferenceConfig(response_prefix="5 + 10 =")
+    resp = llm.prompt(
+        [LLMMessage(content="Output the sum of 5 and 10 without additional explanation")], config
+    )
+    assert resp == LLMMessage(role="assistant", content=[TextContent(text="5 + 10 =  15")])
+
+
+def test_json_mode() -> None:
+    json_mode_prompt = """
+        Create a list of the numbers 1 through 5.
+
+        Here's an example of the desired output for the number 2 through 6
+        {"result": [2, 3, 4, 5, 6]}
+    """
```

Comment:
This doesn't seem to make a difference here, but note that due to the way Python's `"""` strings work, the indentation of the lines here is included as whitespace in the string.

One way to avoid that (that admittedly doesn't look very nice) is to get rid of the indentation like so:
```py
    json_mode_prompt = """\
Create a list of the numbers 1 through 5.
Here's an example of the desired output for the number 2 through 6
{"result": [2, 3, 4, 5, 6]}
"""
```

Reply:
You don't need to change this, just making a note.